.gitignore

```diff
@@ -264,3 +264,4 @@ deployment/**/pkg-dist-tmp/
 deployment/collect-dist/
 
 jellyfin_version.ini
+MediaBrowser.WebDashboard/jellyfin-web
```

Comment:
uhh, WAT?!

Reply:
Yea this should definitely not be `.gitignore`'d, the submodule is critical to the build process.

Reply:
Likely reason for this:

If you update the web ui yourself, it modifies the submodule file and tries to have it committed.

I just commit all changed EXCEPT the submodule manually since gitignoring it will produce problems.

Reply:
Yeah this can go, I'll put in a commit this evening to revert it. Was trying to keep it from preventing me from using git commit -a

Reply:
`git commit -u` is your friend 😉